doc/reference/configuration/configuration_reference.rst

```diff
@@ -221,6 +488,718 @@ This section describes options related to :ref:`storing configuration in etcd <c
+
+.. confval:: replication.bootstrap_strategy
+
+    Specifies a strategy used to bootstrap a :ref:`replica set <replication-bootstrap>`.
+    The following strategies are available:
+
+    *   ``auto``: a node doesn't boot if half or more of the other nodes in a replica set are not connected.
+        For example, if a replica set contains 2 or 3 nodes, a node requires 2 connected instances.
+        In the case of 4 or 5 nodes, at least 3 connected instances are required.
+        Moreover, a bootstrap leader fails to boot unless every connected node has chosen it as a bootstrap leader.
+
+    *   ``config``: use the specified node to bootstrap a replica set.
+        To specify the bootstrap leader, use the :ref:`<replicaset_name>.bootstrap_leader <configuration_reference_replicasets_name_bootstrap_leader>` option.
+
+    *   ``supervised``: a bootstrap leader isn't chosen automatically but should be appointed using ``box.ctl.make_bootstrap_leader()`` on the desired node.
+
+        Configuration fails if no bootstrap leader is appointed during a :ref:`replication.connect_timeout <configuration_reference_replication_connect_timeout>`.
+
+    *   ``legacy`` (deprecated since :doc:`2.11.0 </release/2.11.0>`): a node requires the :ref:`replication_connect_quorum <cfg_replication-replication_connect_quorum>` number of other nodes to be connected.
+        This option is added to keep the compatibility with the current versions of Cartridge and might be removed in the future.
+
+    | Type: string
+    | Default: ``auto``
+    | Environment variable: TT_REPLICATION_BOOTSTRAP_STRATEGY
+
+
+.. _configuration_reference_replication_connect_timeout:
+
+.. confval:: replication.connect_timeout
+
+    A timeout (in seconds) a replica waits when trying to connect to a master in a cluster.
+    See :ref:`orphan status <replication-orphan_status>` for details.
+
+    This parameter is different from
+    :ref:`replication.timeout <configuration_reference_replication_timeout>`,
+    which a master uses to disconnect a replica when the master
+    receives no acknowledgments of heartbeat messages.
+
+    | Type: number
+    | Default: 30
+    | Environment variable: TT_REPLICATION_CONNECT_TIMEOUT
+
+
+.. _configuration_reference_replication_election_mode:
+
+.. confval:: replication.election_mode
+
+    A role of a replica set node in the :ref:`leader election process <repl_leader_elect>`.
+
+    The possible values are:
+
+    *   ``off``
+
+        A node doesn't participate in the election activities.
+
+    *   ``voter``
+
+        A node can participate in the election process but can't be a leader.
+
+    *   ``candidate``
+
+        A node should be able to become a leader.
+
+    *   ``manual``
+
+        Allow to control which instance is the leader explicitly instead of relying on automated leader election.
+
+        By default, the instance acts like a voter -- it is read-only and may vote for other candidate instances.
```

Comment:
`replication.election_mode` is effectively ignored if `failover` is set to something other than `election`. And it is `candidate` by default if `failover: election` is enabled. See [here](https://github.com/tarantool/tarantool/blob/2c422a0b98675cab480ebb44256dfeb72bd2f820/src/box/lua/config/applier/box_cfg.lua#L376-L380).

Reply:
Ouch, got it. It is about `replication.election_mode: manual`. I interpreted this paragraph as if it is about the `replication.election_mode` itself.

Anyway, the information above is missing here and it worth to include it. The default is marked as `box.NULL`. It is technically correct (`config:get('replication.election_mode')` will return this value), but the effective default depends on `replication.failover`. Such effective defaults are usually marked in the config schema code, see [here](https://github.com/tarantool/tarantool/blob/2c422a0b98675cab480ebb44256dfeb72bd2f820/src/box/lua/config/instance_config.lua#L1181-L1183).